src/service/url-config-service.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (config) {
 
         host: (function () {
             var host = window.location.host;
-            if (!host || host.indexOf('localhost') !== -1) {
+            if (!host || host.indexOf('local.forio') !== -1) {
                 host = 'forio.com';
```

Comment:
we still need the host to be forio.com if its localhost, for most of the api calls, so how about host.indexOf('local')